code/modules/hippie/sound/pooling.dm

```diff
@@ -0,0 +1,145 @@
+#define POOL_HIT_COUNT 1
```

Comment:
This entire file is **bad**

- Adding this would add pooling for other things as well, please nuke all the random procs, defines and other garbage from this file and turn this into a pooling system for `/sound` only and only `/sound`